resources/js/components/FormField.vue

```diff
@@ -106,12 +106,30 @@ export default {
             }).authorizedToCreate;
         },
     },
+
     beforeDestroy() {
-        Nova.$off("nova-belongsto-depend-" + this.field.dependsOn);
+      if (this.field.dependsOn) {
```

Comment:
This can and should support both string and array inputs for dependsOn.

```js
if (typeof this.field.dependsOn === 'string') {
  Nova.$off(`nova-belongsto-depend-${this.field.dependsOn}`);
} else if (this.fields.dependsOn instanceof Array) {
  // incoming changes
}
```